techniques/systemSettings/misc/clockConfiguration/3.0/changelog

```diff
@@ -4,3 +4,7 @@
  -- Benoît Peccatte <benoit.peccatte@normation.com> Fri Oct 17 14:10:43 CEST 2014
   * Version 3.0
   ** Use rudder_common_report instead of reports:
+ -- Janos Mattyasovszky <mail@matya.hu> Sun Jan 31 01:04:00 CET 2016
+  * Version 3.0
```

Comment:
Did not find any detailed information about when to increment the minor version.. . 
Basically no new stuff was added in the means of features, only the representation in the gui was changed. Should I make it 3.1 then?